src/Init/Conv.lean

```diff
@@ -54,7 +54,7 @@ macro "left" : conv => `(lhs)
 macro "right" : conv => `(rhs)
 macro "intro " xs:(colGt ident)* : conv => `(ext $xs*)
 
-syntax enterArg := ident <|> group("@"? num)
+syntax enterArg := ident <|> ("@"? num)
```

Comment:
This is a separate pitfall: `<|>` is an operator on two `stx`, not `stx+`. I think we would need to change all occurrences of `stx+` to `sepBy1(stx+, " <|> ")` to fix this.